apps/home-dashboard/src/App.tsx

```diff
@@ -1,5 +1,5 @@
 import "./App.css"
-import { Button } from "ui/Button"
+import { Button } from "ui/src/components/Button/Button"
```

Comment:
In the future, I would like to move this under a namespace like '@ui/Button/Button'